api/krusty/configmaps_test.go

```diff
@@ -291,6 +291,8 @@ configMapGenerator:
     behavior: merge
     literals:
     - ANOTHER_ENV_VARIABLE="bar"
+    options:
+      disableNameSuffixHash: true
```

Comment:
I had to add this option to get the same output in the test. I'm not sure why it wasn't adding a hash suffix before?

Reply:
This could be a bug fix, but I'm really not sure, and it bothers me that I don't understand why this is happening (though I have a theory now, posted in another comment). The test is showing a situation where the ConfigMap generator is essentially generating an overlay for a user-provided resource that would not otherwise get suffixed. It's not obvious to me which behavior is correct.

Reply:
Your theory seems correct, the test is now as it was before!